frontend/src/scenes/insights/InsightTabs/FunnelTab/FunnelCorrelationTable.tsx

```diff
@@ -88,27 +96,41 @@ export function FunnelCorrelationTable(): JSX.Element | null {
         )
     }
 
-    const parseEventAndProperty = (item: string): { name: string; property?: PropertyFilter } => {
-        const components = item.split('::')
+    const parseEventAndProperty = (
```

Comment:
non-blocking

Can we put a summary of what the structure here is or something. If I'm someone just coming to this code I have no idea what this is meant to be.

At some point it might be nice to actually define a structure the rest api can return that captures the various different things it can return?

Reply:
Agreed!

Reply:
This looks eerily 👻  like it's some type of API result annotation that could go in a selector.

Reply:
Haa, yes, this makes sense. It's getting complicated enough now 🤦‍♂️ . Used to be just a `property::value`.